tests/test_lab_dev/test_transformations/test_ggate.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Test for the Ggate class.
+"""
+
+from mrmustard.lab_dev import Ggate, Identity
+from mrmustard import math
+
+
+def test_ggate_is_unitary_1mode():
```

Comment:
Can we change this to match our other unit tests? As in, 
```
TestGgate:
r"""
"""

def test_init()
...

def test_ggate_is_unitary_1mode()
...
```
You can you the other test files as an example

Reply:
Sure, will do.